### PR TITLE
MOCA-2353: Oauth flow readme update

### PR DESCRIPTION
**Pull request by hartmannj**
…and content type

### CODE REVIEW

File: oauth2.md [hunk 1]
```diff
@@ -148,10 +148,26 @@ The above example contains a request with the following parameters
...
+
+The access token is a JWT token that contains the following fields:
+```json
+{
+  "realmId": "random.moca",
+  "userId": "6ddc2453-2172-4ba4-b8b0-afade2cdbd64",
+  "partnerId": "421c7962-f04c-4429-b86b-2fc140a001e6",
+  "iat": 1714369562,
+  "exp": 1714455962
+}
```
Comment (abcalebh) [inline oauth2.md]: do we want to publicize this info?

File: oauth2.md [hunk 2]
```diff
@@ -165,7 +181,7 @@ Link a user id of your platform to Mocaverse.
 #### Request
 
 ```
-POST https://api.moca-id.mocaverse.xyz/api/oauth/partner/user
+POST https://oauth.mocaverse.xyz/api/oauth/partner/user
```
Comment (abcalebh) [inline oauth2.md]: think this remains as `api.moca-id`.

oauth is just to get the token

Comment (hartmannj) [reply oauth2.md]: For both request changes?

File: oauth2.md (hunk 1 above)
Comment (hartmannj) [reply oauth2.md]: I mean it is a jwt, public by design. 😅
We can also just mention that the token contains realmId and userId without explicitly showing the payload here?

Comment (abcalebh) [reply oauth2.md]: not sure if this payload would change. I think we can say realmId would be there

File: oauth2.md (hunk 2 above)
Comment (abcalebh) [reply oauth2.md]: yes, all API requests

**Review by abcalebh — APPROVED** [review]

**Pull request merged**